data/zilliqa/assets/zil1vdc79fg75sth96xnt6xmq5f55swcd5t6m5zhvn/meta.json

```diff
@@ -1,6 +1,18 @@
 {
   "name": "YODA",
   "symbol": "YODA",
+  "web": "http://jody.tilda.ws",
+  "whitepaper": "http://jody.tilda.ws/whitepaper",
+  "supply": 10000,
+  "decimals": 5,
+  "email": "anzellkennard35@gmail.com",
+  "publicTeam": true,
+  "links": {
+    "twitter": "https://twitter.com/Jody96718451",
+    "telegram": "https://t.me/Yodaco",
+    "linkedin": "https://www.linkedin.com/in/anzell-kennard-87817021b",
```

Comment:
this field is for linkedin companies that should include at least one public person. right now this profile doesn't even say he works for the yoda token.

Reply:
really? there are many examples of personal profiles - Its my personal project.

Reply:
They list their affiliation though. Otherwise mind as well say I'm jeff bezos on linkedin launching yoda token

Reply:
cryptometa/data/zilliqa/assets/zil108ex3na3dvwnjq5p4nls4z3gjy8fx66693hf4d/meta.json

this is for consult token.. this linkedin doesnt mentioned consult?

Reply:
Keep wasting your time trying to point out stuff you think is the same. This won't get merged as is.

Reply:
Yep doing that now.. edit on the linkedin site coming soon.

Reply:
Hi Balthazar, unfortunately the requirements for a LinkedIn company page is far too great for my small project, it requires an email address with a company domain name, and several other things.  I will try and get them over time however.

I have updated my linkedin profile to show that Im the creator, I was wondering if this will suffice? It proves im the owner and that ive edited that page.